src/main/java/org/broadinstitute/barclay/argparser/PositionalArgumentDefinition.java

```diff
@@ -110,6 +110,60 @@ public void validateValues(final CommandLineArgumentParser commandLineArgumentPa
         }
     }
 
+    @Override
+    public String getDocString() { return positionalArgumentsAnnotation.doc(); }
+
+    /**
+     * Return a string with the usage statement for this positional argument.
+     * @param argumentColumnWidth width reserved for argument name column display
+     * @param descriptionColumnWidth width reserved for argument description column display
+     * @return the usage string for this argument
+     */
+    public String getArgumentUsage(final int argumentColumnWidth, final int descriptionColumnWidth) {
+
+        final StringBuilder sb = new StringBuilder();
+        sb.append("--").append("POSITIONAL");
```

Comment:
Should this include which position it should be in?  Is that easy to find?  I

Reply:
Not sure how to convey it, but positional args must always be first. But these are hardly ever used.

Reply:
I updated this String to say `"POSITIONAL (must be first)"`.